pegasus/src/database.rb

```diff
@@ -5,10 +5,11 @@ class Tutorials
 
   def initialize(table)
     @table = table
+    @contents = DB[@table].all
   end
 
   def launch_url_for(code,domain)
-    return DB[:beyond_tutorials].where(code: code).first[:url] if @table == :beyond_tutorials
+    return @contents.find {|row| row[:code] == code}[:url] if @table == :beyond_tutorials
```

Comment:
Is this one still used?
https://github.com/code-dot-org/code-dot-org/blob/staging/pegasus/cache/Data_UkHocTutorials.gsheet.csv


On Mon, Dec 14, 2015 at 2:41 PM, Brendan Reville <notifications@github.com>
wrote:

> In pegasus/src/database.rb
> <https://github.com/code-dot-org/code-dot-org/pull/6053#discussion_r47570405>
> :
>
> >    end
> >
> >    def launch_url_for(code,domain)
> > -    return DB[:beyond_tutorials].where(code: code).first[:url] if @table == :beyond_tutorials
> > +    return @contents.find {|row| row[:code] == code}[:url] if @table == :beyond_tutorials
>
> :tutorials
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/code-dot-org/code-dot-org/pull/6053/files#r47570405>.
>